host/cluster-down.yaml

```diff
@@ -0,0 +1,306 @@
+# Spec to run when a kURL cluster is down and in-cluster specs can't be run
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: cluster-down
+spec:
+  hostCollectors:
+    # System Info Collectors
+    - blockDevices: {}
+    - cpu: {}
+    - hostOS: {}
+    - hostServices: {}
+    - ipv4Interfaces: {}
+    - memory: {}
+    - time: {}
+    # Certificate Info for ETCD and K8s API
+    - certificate:
+        collectorName: k8s-api-keypair
+        certificatePath: /etc/kubernetes/pki/apiserver.crt
+        keyPath: /etc/kubernetes/pki/apiserver.key
+    - certificate:
+        collectorName: etcd-keypair
+        certificatePath: /etc/kubernetes/pki/etcd/server.crt
+        keyPath: /etc/kubernetes/pki/etcd/server.key
+    # Disk usage for commonly used directories in kURL installs
+    - diskUsage:
+        collectorName: root
+        path: /
+    - diskUsage:
+        collectorName: var-lib-kubelet
+        path: /var/lib/kubelet
+    - diskUsage:
+        collectorName: var-lib-docker
+        path: /var/lib/docker
+    - diskUsage:
+        collectorName: var-lib-containerd
+        path: /var/lib/containerd
+    - diskUsage:
+        collectorName: var-lib-rook
+        path: /var/lib/rook
+    - diskUsage:
+        collectorName: opt-replicated
+        path: /opt/replicated
+    - diskUsage:
+        collectorName: var-openebs
+        path: /var/openebs
+    - http:
+        collectorName: curl-k8s-api-6443
+        get:
+          url: https://localhost:6443
+          insecureSkipVerify: true
+    # Run collectors for system information
+    - run:
+        collectorName: "uptime"
+        command: "uptime"
+        args: []
+    - run:
+        collectorName: "uname"
+        command: "uname"
+        args: ["-a"]
+    - run: 
+        collectorName: "df"
+        command: "df"
+        args: ["-h"]
+    - run: 
+        collectorName: "iostat"
+        command: "iostat"
+        args: []
+    - run: 
+        collectorName: "sestatus"
+        command: "sestatus"
+        args: []
+    - run: 
+        collectorName: "apparmor-status"
+        command: "apparmor_status"
+        args: []
+    - run: 
+        collectorName: "docker-info"
+        command: "docker"
+        args: ["info"]
+    - run: 
+        collectorName: "crictl-info"
+        command: "crictl"
+        args: ["info"]
+    - run: 
+        collectorName: "crictl-ps"
+        command: "crictl"
+        args: ["ps", "-a"]
+    - run: 
+        collectorName: "docker-ps"
+        command: "docker"
+        args: ["ps", "-a"]
+    - run: 
+        collectorName: "iptables"
+        command: "iptables"
+        args: ["-L", "-v"]
```

Comment:
can we get `lsblk` and `cat /etc/*release` also?

Reply:
These are covered by the `blockDevices` and `hostOS` host collectors.